proposals/sandbox/flux.md

```diff
@@ -151,13 +151,13 @@ In the [GitHub project](https://github.com/weaveworks/flux).
 ## Website
 
 None at present; documentation is in markdown format in the [GitHub
-repo](https://github.com/weaveworks/flux/tree/master/site).
+repo](https://github.com/weaveworks/flux/tree/master/docs).
```

Comment:
why not (`https://github.com/fluxcd/flux/blob/master/docs/index.md/`)

Reply:
Because the link points to where the documentation lives, e.g. which sub-directory.

_It moved to from `./site` to `./docs` and is now available on `docs.fluxcd.io`._

Reply:
I understand. I just thought we would save the redirect that happens;

```
sdake@beast-01:~$ curl https://github.com/weaveworks/flux/tree/master/docs/
<html><body>You are being <a href="https://github.com/fluxcd/flux/tree/master/docs">redirected</a>.</body></html>
```

As this hits the website multiple times.